elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -30,73 +14,59 @@ div.pl-order-blocks-bottom {
 }
 
 
-ul.dropzone {
-  /* This style applies to answer dropzone */
-  background-color: #FFA; /* Yellow */
-  min-height: 30px;
+ul.pl-order-blocks-connected-sortable li:hover{
+  cursor: move; /* For UX: The answer tiles shows a 'grab' cursor on mouse over */
+  outline: 2px solid blue;
+  outline-offset: -2px;
 }
 
-
-ul.pl-order-blocks-connected-sortable li, ul.dropzone li{
-  /* This style applies to the answer draggable tiles themselves */
-  background-color: #EFEFEF; /* Grey */
-  border: 2px solid #A9A9A9;
-  border-radius: 4px;
-  margin: 0 10px 10px 10px;
-  padding: 5px;
-  font-size: 1.2em;
-
-  display: block;
-
-  /* Show overflow text in the answer draggable tiles */
-  white-space: normal;
-  height: auto;
-  max-height: none;
-  overflow: visible;
+li.ui-state-highlight { 
+  /*This styling applies to the placeholder answer tile that appears when the user is dragging */
+  height: 3em !important; 
+  border: none !important;
+  list-style-type: none;
+  background-color: #90a4ae !important;
+  min-width: 3em !important;
 }
 
-ul.pl-order-blocks-connected-sortable li:hover, ul.dropzone li:hover{
-  cursor: move; /* For UX: The answer tiles shows a 'grab' cursor on mouse over */
+ul.header {
+  color: black;
+  list-style-type: none;
+  margin-bottom: 0px;
 }
 
+ul.inline li {
+  display: inline-block;
+  margin-right: 5px;
+}
 
-li.info {
-  /* This styling applies to the instruction text for the dropzones */
-  background-color: transparent !important;
-  border: none !important;
-  cursor: auto !important;
+ul.ui-sortable {
+  padding: 17px 17px 17px 17px;
+  height: 100%;
+  margin-bottom: 0px;
 }
 
-li.ui-state-highlight { 
-  /*This styling applies to the placeholder answer tile that appears when the user is dragging */
-  height: 2em !important; 
-  line-height: 1.5em !important;
-  border: none !important;
-  color: #efefff !important; 
-  background-color: #efefff !important; 
+ul.ui-sortable li, ul.answer_table li, ul.submission_table li {
+  margin-bottom: 5px;
+  border: 1px solid grey !important;
+  border-radius: 5px !important;
+  line-height: 1.1em;
```

Comment:
I did a little more playing around with the font-size/line-height issue. If you just change this to 1.2em, it helps with the text not being too crowded in multi-line blocks, but without having the blocks get too big (as they were before when line-height and font-size were higher).

Reply:
Yep, implemented your suggestion